core/src/main/java/io/grpc/transport/AbstractStream.java

```diff
@@ -348,7 +348,7 @@ protected final void onSendingBytes(int numBytes) {
    *
    * @param numBytes the number of bytes that were sent.
    */
-  protected final void onSentBytes(int numBytes) {
+  protected void onSentBytes(int numBytes) {
```

Comment:
Done but I need to make it public in AbstractStream so it can be called by the handlers.